src/shogun/kernel/SpectralMixtureKernel.cpp

```diff
@@ -0,0 +1,221 @@
+/*
+ * This software is distributed under BSD Clause 3 license (see LICENSE file).
+ *
+ * Written (W) 2015 Esben Soerig
+ */
+
+#include <shogun/kernel/SpectralMixtureKernel.h>
+
+using namespace shogun;
+
+CSpectralMixtureKernel::CSpectralMixtureKernel() : CDotKernel()
+{
+	/* We can't really say anything about number of components at this point
+	   We have to leave the object in an unusable state until a set of feature
```

Comment:
thats ok and standard. Will be catched by the REQUIRE statements in the later code